iot_config/src/route_service.rs

```diff
@@ -309,6 +309,10 @@ impl iot_config::Route for RouteService {
                 }
             }
 
+            drop(active_routes);
```

Comment:
i have tested this and received the desired result (all initial updates are streamed out followed by ongoing updates as they're processed by the config service) but particularly looking to @maplant or @madninja to confirm my suspicion that this should release the hold on the database pool by the long-running subscriber threads (the packet routers) so they're not continuously tying up a pool connection they'll never use after processing the initial set of results each time a packet router connects